### ISSUE TITLE
Is there a way to ignore a workspace member crate?

### ISSUE THREAD

**Issue by frederikhors** [bnjbvr/cargo-machete#107]
Is there a way to ignore check in a workspace member crate?

**Comment by bnjbvr** [comment]
Nope; we are adding a similar mechanism in https://github.com/bnjbvr/cargo-machete/pull/95, but what you're asking is slightly different. Would you be interested in working on this?

**Comment by frederikhors** [comment]
I don't know where to start.
